<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoPredictionDialog</class>
 <widget class="QDialog" name="DoPredictionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1628</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="1" column="2">
    <widget class="QGroupBox" name="dataGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QListView" name="listView_3">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QFrame" name="prednameFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Prediction Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="predname"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="projectGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Projects</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListView" name="listView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="modelGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Models</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListView" name="listView_2">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QGroupBox" name="objectGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Objects</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" colspan="2">
       <widget class="QListView" name="listView_4">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="objSelectAllButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="objInvertSelectionButton">
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Invert Selection</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="objSelectByLabel">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Select by label...</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="objUnselectButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Unselect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QGroupBox" name="YvariableGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Y Variables</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="3" column="1">
       <widget class="QPushButton" name="yvarInvertSelectionButton">
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Invert Selection</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QListView" name="listView_5">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="yvarSelectAllButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QComboBox" name="yvarSelectByLabel">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Select by label...</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="yvarUnselectButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Unselect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Import target</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="2">
      <widget class="QPushButton" name="nextButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="prevButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
